README.md

```diff
@@ -1,310 +1,320 @@
+
+  ```bash
+  # Set the localtion to cuGraph in an environment variable CUGRAPH_HOME 
+  export CUGRAPH_HOME=$(pwd)/cugraph
+
+  # Download the cuGraph repo
+  git clone https://github.com/rapidsai/cugraph.git $CUGRAPH_HOME
+
+  # Next load all the submodules
+  cd $CUGRAPH_HOME
+  git submodule update --init --remote --recursive
+  ```
+
+
+
+2) Create the conda development environment 
+
+​	A)   Building the `master` branch uses the `cugraph_dev` environment
+
+```bash
+# create the conda environment (assuming in base `cugraph` directory)
+# for CUDA 9.2
+conda env create --name cugraph_dev --file conda/environments/cugraph_dev.yml
+
+# for CUDA 10
+conda env create --name cugraph_dev --file conda/environments/cugraph_dev_cuda10.yml
+
+# activate the environment
+conda activate cugraph_dev 
+
+# to deactivate an environment
+conda deactivate
+```
+
+
+
+​	B) Create the conda development environment `cugraph_nightly`  
+
+If you are  on the latest development branch then you must use the `cugraph_nightly` environment.  The latest cuGraph code uses the latest cuDF features that might not yet be in the master branch.  To work off of the latest development branch, which could be unstable, use the nightly build environment.  
+
+```bash
+# create the conda environment (assuming in base `cugraph` directory)
+conda env create --name cugraph_nightly --file conda/environments/cugraph_nightly.yml
+
+# activate the environment
+conda activate cugraph_nightly 
+
+```
+
+
+  - The environment can be updated as development includes/changes the dependencies. To do so, run:  
+
+    ```bash
+    # for CUDA 9.2
+    conda env update --name cugraph_dev --file conda/environments/cugraph_dev.yml
+    
+    conda activate cugraph_dev 
+    ```
+
+
+
+3) Build and install `libcugraph`. CMake depends on the `nvcc` executable being on your path or defined in `$CUDACXX`.
+
+  This project uses cmake for building the C/C++ library. To configure cmake, run:
+
+  ```bash
+  cd $CUGRAPH_HOME
+  cd cpp	      		# enter cpp directory
+  mkdir build   		# create build directory 
+  cd build     		# enter the build directory
+  cmake .. -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX 
+
+  # now build the code
+  make -j				# "-j" starts multiple threads
+  make install		# install the libraries 
+  ```
+
+The default installation  locations are `$CMAKE_INSTALL_PREFIX/lib` and `$CMAKE_INSTALL_PREFIX/include/cugraph` respectively.
+
+
+
+#### Building and installing the Python package
+
+5. Install the Python package to your Python path:
+
+```bash
+cd $CUGRAPH_HOME
+cd python
+python setup.py install    # install cugraph python bindings
+```
+
+
+
+
+
+#### Run tests
+
+6. Run either the standalone tests or the Python tests with datasets
+  - **C++ stand alone tests** 
+
+    From the build directory : `gtests/gdfgraph_test`
+
+    ```bash
+    # Run the cugraph tests
+    cd $CUGRAPH_HOME
+    cd cpp/build
+    gtests/GDFGRAPH_TEST		# this is an executable file

```

Comment:
I typically run make test at this point.  make test runs ALL of the unit tests.  tests/GDFGRAPH_TEST only tests the C++ side of the python to C++ bindings.

Reply:
I can replace gtests/GDFGRAPH_TEST with make test